publisher/publisher.go

```diff
@@ -0,0 +1,57 @@
+package publisher
+
+import (
+	"fmt"
+
+	"github.com/screwdriver-cd/sd-cmd/config"
+	"github.com/screwdriver-cd/sd-cmd/screwdriver/api"
+	"github.com/screwdriver-cd/sd-cmd/util"
+)
+
+func init() {
+	config.LoadConfig()
```

Comment:
I think we do not need call this one here, because sd-cmd.go call this method.

Reply:
You're right. I've deleted it.